docker-compose.yml

```diff
@@ -0,0 +1,40 @@
+services:
+  postgres:
+    image: postgres:latest
+    volumes:
+      - db_data:/var/lib/postgresql/data
+    environment:
+      - POSTGRES_DB=postgres
+      - POSTGRES_USER=postgres
+      - POSTGRES_PASSWORD=postgres
+    ports:
+      - "5432:5432"
+    networks:
+      - default
+    deploy:
+      resources:
+        limits:
+          cpus: "2"
+          memory: 2G
+      restart_policy:
+        condition: on-failure
+
+  thirdweb:
+    image: thirdweb/engine:latest
```

Comment:
We can use `latest`. But, I would add notes here, to always use a versioned image instead of `latest` as it helps in understanding which version worked and which the user had issue on

Reply:
I agree, I'll leave a note and a note in the README too.

Reply:
I used the versioned version and add note to both readme and on the docker compose file.